packages/backend/src/tokens/interstitial.ts

```diff
@@ -20,6 +20,14 @@ export type LoadInterstitialOptions = {
   domain?: string;
 };
 
+// TODO: use the same function from @clerk/shared once treeshakable
+function addClerkPrefix(str: string | undefined) {
```

Comment:
🔧 This function can only return an empty string if str is not provided. That would create a simpler type.

Reply:
I'd also suggest making it bullet prof and ensuring that `clerk.` is added only once via regex replacement ;). That is

```
addClerkProfix("") -> ""
addClerkProfix("foo.com") -> "clerk.foo.com"
addClerkProfix("clerk.foo.com") -> "clerk.foo.com"
```

Reply:
i agree but wouldn't our use case would make the regex complicated?
eg
```
addClerkProfix("clerk.clerk.dev") -> "clerk.clerk.dev"
addClerkProfix("clerk.dev") -> "clerk.clerk.dev"
```

Reply:
@SokratisVidros  should we handle the case that @dimkl  mentioned  ?

Reply:
Yes.

Reply:
@SokratisVidros you can see the new cases in our [tests](https://github.com/clerkinc/javascript/pull/866/commits/14cd4008ed6ba91bccfd236c450ff9b247983212#diff-1a43d1aabd7f12b0efe74b95add43652387ef9b16e70fa0ba015a63b72c78771L34)

I think we are handling everything now